docs/changelog/105271.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 105271
+summary: Efficiently encode multi-valued dimensions
+area: TSDB
+type: enhancement
```

Comment:
maybe label this issue as a non-issue? It improves #99747, which hasn't been released yet. So an entry in the changelog about could be confusing.